### PR TITLE
SORA Account Security best practices

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

File: src/account-security.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    - name: twitter:site
+      content: "@sora_xor"
+  - - meta
+    - name: twitter:creator
+      content: "@sora_xor"
+---
+
+# SORA Account Security: How to Manage Passphrases Securely
+
+#### Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today.
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today.
```

File: src/account-security.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- Employ best practices such as using unique passwords, 2FA, and opting for physical and digital encrypted storage methods.
+- Be sure to regularly review and update your security methods to stay ahead of evolving cyber threats.
+
+In today's digital age, securing one's digital assets has become paramount. The linchpin of this security often revolves around the management of passphrases, passwords, and other digital keys. Understanding the different storage formats and following best practices can distinguish between secure accounts and potential breaches.
+
+![](.gitbook/assets/sora-account-security.png)
+
+## Mnemonic, JSON, and Raw Seed: An Overview
+
+- **Mnemonic Passphrases:** These are words that can be used to derive a cryptographic key. They're typically easier to remember than raw cryptographic keys and are often employed by crypto wallets to allow users to recover their accounts.
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
- **Mnemonic Passphrases:** A mnemonic phrase is a collection of words that can be used to derive a cryptographic key. Such phrases are typically easier to remember than raw cryptographic keys and are often employed by crypto wallets to allow users to recover their accounts.
```

File: src/account-security.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  - - meta
+    - name: twitter:creator
+      content: "@sora_xor"
+---
+
+# SORA Account Security: How to Manage Passphrases Securely
+
+#### Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today.
+
+## TL;DR:
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
## TL;DR
```

File: src/account-security.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+## Mnemonic, JSON, and Raw Seed: An Overview
+
+- **Mnemonic Passphrases:** These are words that can be used to derive a cryptographic key. They're typically easier to remember than raw cryptographic keys and are often employed by crypto wallets to allow users to recover their accounts.
+
+- **JSON Keystores:** A JSON keystore contains an encrypted version of a user's private key. It's accompanied by other data and is formatted as a JSON object. This keystore requires a password to decrypt and access the private key.
+
+- **Raw Seed:** The raw seed is a direct representation of a cryptographic key. It is typically a string of letters and numbers. It's the most basic and direct form of a key but also the least human-friendly.
+
+## Best Practices to Store Account Passwords and Information:
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
## Best Practices to Store Account Passwords and Information
```

File: src/account-security.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+- **Use Strong, Unique Passwords:** Avoid using the same password for multiple accounts. Make sure they're lengthy, use a mix of characters, numbers, and symbols, and avoid obvious phrases.
+
+- **Regularly Update Passwords:** Changing your passwords periodically can help protect against undetected breaches.
+
+- **Two-Factor Authentication (2FA):** This adds an extra layer of security by requiring a second form of verification and a password.
+
+### Physical Storage Options:
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
### Physical Storage Options
```

File: src/account-security.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+Here are key security recommendations:
+
+- **Database Security:** Prioritize the security of the password manager's database to thwart unauthorized access.
+- **Regular Backups:** Schedule consistent backups of the password manager database, ensuring protection against unforeseen data loss or breaches.
+- **Master Password Strength:** Encrypt the database with a robust and unique master password, making unauthorized decryption challenging.
+- **Safe Downloads:** Always download password keeper software from recognized, legitimate sources to avoid inadvertently acquiring malicious software.
+- **Opt for Reputed Software:** Choose password managers that have garnered solid reputations and positive reviews, ensuring both their reliability and their commitment to user security.
+
+### Social Engineering Awareness:
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
### Social Engineering Awareness
```

File: src/account-security.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+Most likely, brute force will not be how your passphrases are compromised. Scams such as phishing (using fake or too-good-to-be-true offers) are the most effective to separate people from their assets. Scams such as popular NFT mints at a bargain or impersonating online service providers can lead to painful losses.
+It is important to ensure that the links you click in emails or tweets are from verified accounts. Don’t be blinded by ambition.
+
+Likewise, other people may also reach out to you, impersonating wallet providers or token teams requesting your mnemonic. No project or infrastructure will ever need your private key; if anyone asks for it, consider this a red flag.
+
+### Linking to Your Stored Data:
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
### Linking to Your Stored Data
```

File: src/account-security.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+When creating user interfaces (UIs) or tutorial articles, always link back to the secure storage solution being used. This ensures that users have a direct path to their stored data, reducing the risk of misplacement or accidental deletion.
+
+In the UI itself, a button or link labelled "Access Stored Passphrases" can be an effective way to guide users. In tutorial articles, embed hyperlinks within the text or clear call-outs to direct readers to the right resources.
+
+## Conclusion
+
+Managing and storing passphrases securely is the cornerstone of digital security. By understanding the different types of storage formats and following best practices, you can protect your digital assets and enjoy peace of mind. Remember, as technology evolves, it's essential to stay updated with the latest security protocols and methodologies.
+
+**Note: Always consult with cybersecurity experts when considering a solution for storing sensitive information.**
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
::: tip
Always consult with cybersecurity experts when considering a solution for storing sensitive information.
:::
```

File: src/account-security.md (hunk 1 above)
Comment (bgodlin) [inline src/account-security.md]: I saw multiple occurrences of British English in this repository. Should we consider changing all of them to conform to it?

Comment (outoftardis) [reply src/account-security.md]: Yes, we should be consistent

**Review by outoftardis — APPROVED** [review]

File: src/account-security.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+::: tip
+Always consult with cybersecurity experts when considering a solution for storing sensitive information.
+:::
+
+---
+
+## Learn More
+
+- [Create an Address](./create-an-address)
+- [Connect Your Wallet to Polkaswap](/polkaswap-connect-wallet.html.md)
```
Comment (outoftardis) [inline src/account-security.md]:
```suggestion
- [Connect Your Wallet to Polkaswap](/polkaswap-connect-wallet.md)
```

**Review by outoftardis — APPROVED** [review]

File: src/account-security.md (hunk 1 above)
Comment (WRRicht3r) [reply src/account-security.md]: Yes, let's keep it British, mate

**Review by bgodlin — APPROVED** [review]
